cvcl/templates/course_detail.html

```diff
@@ -1,9 +1,34 @@
 {% extends "master.html" %}
+{% load bootstrap3 %}
 
 {% block content %}
     <h1>Course: {{ course.name }}</h1>
-
-    {% if user.is_instructor %}
-        TODO: list of user
-    {% endif %}
-{% endblock %}
+    <table class="table table-hover">
+        <tr>
+            <th>Student username</th>
+        </tr>
+        {% if user.is_instructor %}
```

Comment:
You should be checking if the instructor is the instructor of this course. You may have a student who is an instructor in a different course but not this one and this condition would pass.